x/wasm/types/gas_meter.go

```diff
@@ -8,7 +8,7 @@ import (
 
 // Constant gas parameters
 const (
-	GasMultiplier = uint64(100) // Please note that all gas prices returned to the wasmVM engine should have this multiplied
+	GasMultiplier = uint64(140_000_000) // Please note that all gas prices returned to the wasmVM engine should have this multiplied
```

Comment:
why is this `140_000_000` ?

Reply:
Please have a look at the wasmd reference implementation. The wasmvm gas calculations made a major step in the upgrade from 0.16 -> 1.0.0. wasmd v0.27.0 is the version that fully incorporates the new wasmvm. The final gas multiplier  was set to 140,000,000 to convert sdk gas to wasmer gas and vice versa.

https://github.com/CosmWasm/wasmd/pull/628
https://github.com/CosmWasm/wasmd/pull/636